src/pages/CommitDetailPage/CommitDetailPage.tsx

```diff
@@ -58,6 +60,49 @@ const CommitDetailPage: React.FC = () => {
     commitId: commitSha,
   })
 
+  const { setBreadcrumbs, setBaseCrumbs } = useCrumbs()
+  useLayoutEffect(() => {
+    setBaseCrumbs([
```

Comment:
Need to set base crumbs here for the case when a user comes directly to the commit detail page without first hitting the RepoPage. Same deal for pull detail page